apps/condo/domains/subscription/components/SubscriptionContext.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useContext, useEffect, useState, createContext } from 'react'
+import Router, { useRouter } from 'next/router'
+import { ServiceSubscription, SortServiceSubscriptionsBy } from '../../../schema'
+import { Modal, Typography } from 'antd'
+import { useIntl } from '@core/next/intl'
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
+import duration from 'dayjs/plugin/duration'
+import { Button } from '@condo/domains/common/components/Button'
+import { useOrganization } from '@core/next/organization'
+import { useObjects } from '../utils/clientSchema/ServiceSubscription'
+import { get } from 'lodash'
+import { isExpired } from '../utils/helpers'
+import { hasFeature } from '../../common/components/containers/FeatureFlag'
+
+dayjs.extend(relativeTime)
+dayjs.extend(duration)
+
+interface ISubscriptionContext {
+    subscription?: ServiceSubscription
+    daysLeft?: number
+    daysLeftHumanized?: string
+    isExpired?: boolean
+}
+
+const SubscriptionContext = createContext<ISubscriptionContext>({})
+
+export const useServiceSubscriptionContext = () => useContext(SubscriptionContext)
+
+const useServiceSubscriptionLoader = (): ISubscriptionContext => {
+    const { organization } = useOrganization()
+    const { objs } = useObjects({
```

Comment:
it's better to use `ServiceSubscription.useObjects`

Reply:
Changed